### ISSUE TITLE
Remove "Make a Suggestion", duplicates Bug feature

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#502]
```
We should move all feedback into one location... the Issue Tracker.  Then 
change "Report Bug" to "Report Bug or Request Feature or Article"
```

Original issue reported on code.google.com by `sethladd@google.com` on 1 Apr 2011 at 10:06

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 03e1db880134.
```

Original comment by `ericbide...@html5rocks.com` on 1 Jun 2011 at 11:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Checked in
```

Original comment by `ericbide...@gmail.com` on 1 Jun 2011 at 11:55
- Added labels: **Priority-P3**, **Milestone-6**, **Maintenance**
- Removed labels: **Priority-P2**
